README.md

```diff
@@ -14,6 +14,40 @@ programming language).
 currently built using TypeScript on top of [oclif](https://oclif.io/) to speed
 up development of the CLI tool.
 
+## Using with Docker
+
+To run `themis-contract` using Docker, you can build the Docker image for it
+yourself for now (we are working on getting up on Docker Hub):
+
+```bash
+# Clone the latest source
+git clone git@github.com:informalsystems/themis-contract.git
+cd themis-contract
+
+# Build a Docker image from the latest source you've just cloned
+docker build -t themis-contract:latest
+
+# Shows the "help" command for Themis Contract
+docker run --rm -it \
+  themis-contract:latest
+
+# Create a new contract from the specified template in the current directory
+# (assumes that the template.md file is in the current directory)
+docker run --rm -it \
+  -v `pwd`:/contracts \
+  themis-contract:latest \
+  new -t ./template.md
+```
+
+**WARNING**: Running `themis-contract` from within a Docker container has the
+following known limitations:
+
+1. Use of a local Keybase service has not yet been tested (you'd need to mount
+   your local Keybase socket into the Docker container at the appropriate
+   place).
+2. Contracts generated by the `themis-contract new` command will contain the
```

Comment:
It's a little tricky, but probably doable with some time and trial and error (as with most features). I'm not sure it's worth it though right now to put in the effort given the discussions around the v2 prototype.

You'd need to pass your full working directory path on your local machine into the Docker container, which would need to replicate that path inside the container and use that as the working directory.

Simply overriding the path injected into the TOML file won't work because, if the template's at a relative location in your local file system, `themis-contract` running inside the container won't find the template when attempting to compile it.

It won't work at all if the template for the contract is located on your local filesystem but further up the tree than your working directory that you pass into the container.